services/src/test/scala/cromwell/services/metadata/WorkflowQueryParametersSpec.scala

```diff
@@ -142,6 +146,38 @@ class WorkflowQueryParametersSpec extends WordSpec with Matchers {
       }
     }
 
+    "reject bad label regex" in {
+      val badLabelRegex = "0-label-key"
```

Comment:
I don't think `regex` is needed in this test name, and this could just be `badLabelKey`